frontend/src/views/Leaderboard.vue

```diff
@@ -118,52 +117,10 @@ export default Vue.component("show-leaderboard", {
     window.addEventListener("resize", this.handleResize)
     getDataSets(this.$store.getters.authenticationHeader())
       .then((response) => {
-        this.datasets = response.data
-        this.datasets = [
-          {
-            "name": "squad",
-            "skill-type": "extractive-qa",
-            "metric": "squad",
-            "mapping": {
-              "id-column": "id",
-              "question-column": "question",
-              "context-column": "context",
-              "answer-text-column": "answers.text"
-            }
-          }, {
-            "name": "quoref",
-            "skill-type": "extractive-qa",
-            "metric": "squad",
-            "mapping": {
-              "id-column": "id",
-              "question-column": "question",
-              "context-column": "context",
-              "answer-text-column": "answers.text"
-            }
-          }, {
-            "name": "commonsense_qa",
-            "skill-type":" multiple-choice",
-            "metric": "exact_match",
-            "mapping": {
-              "id-column": "id",
-              "question-column": "question",
-              "choices-columns": ["choices.text"],
-              "choices-key-mapping-column": "choices.label",
-              "answer-index-column": "answerKey"
-            }
-          }, {
-            "name": "cosmos_qa",
-            "skill-type": "multiple-choice",
-            "metric": "exact_match",
-            "mapping":{
-              "id-column": "id",
-              "question-column": "question",
-              "choices-columns": ["answer0", "answer1", "answer2", "answer3"],
-              "choices-key-mapping-column": null,
-              "answer-index-column": "label"
-            }
-          }
-        ]
+        for (let item_dataset = 0; item_dataset < response.data.length; item_dataset++){
```

Comment:
Nein. Bitte nicht.
Du musst hier nur Zeile 122 löschen.
Also quasi, dass es ist:
```js
.then((response) => {
  this.datasets = response.data
}
```

fertig. mehr nicht

Reply:
Wie iteriert Du bitte die Namen der Datasets? die GetdatSet () Methode gibt mehrere Einträge zurück. Wie iteriert Du die Einträge, ohne For-Schleife. Ich habe eingebau leider das funktioniert nicht mehr.

Reply:
Das hat ja vorher auch geklappt. Da waren ja die gleichen Daten wie jetzt. Nur halt dass die statisch waren und nicht über den Endpunkt gekommen sind (aber ja auch schon Objekte und nicht nur Namen).

Hier wird aus den Objekten => Namen gemacht. 
```js
computed: {
    datasetNames: function() {
      return this.datasets.map(dataset => dataset.name);
    }
  },
```
Und das wird ja bei :options benutzt in dem select dann `:options=="datasetNames"`

Reply:
ok, stimmt so kann man auch schreiben. ich  verstehe jetzt was Du meinst. Ich mache die Anpassung schnell wie möglich! 
Danke
Aber das Element v-model="value", sonst das funktioniert nicht mehr.